examples/vertex_ai_orchestration/README.md

```diff
@@ -0,0 +1,148 @@
+an easy way to quickly run your code in a production ready, repeatable 
+cloud orchestrator that requires minimal setup without provisioning and paying 
+for standby compute. 
+
+## 📄 Pre-requisites
+
+In order to run on Vertex AI you will need to do quite a bit of configuration
+within GCP to create all the required resources. In sum you will need to have a
+[GCP container registry](https://cloud.google.com/container-registry/docs), a
+[GCP bucket](https://cloud.google.com/storage/docs/creating-buckets),
+[CloudSQL database](https://cloud.google.com/sql/docs/mysql/create-instance) and
+a [GCP Secret Manager](https://cloud.google.com/secret-manager). Additionally,
+the [Vertex AI API](https://cloud.google.com/vertex-ai/?hl=en_GB&_ga=2.241201409.-205697788.1651483076)
+needs to be enabled.
+
+On top of having these resources you will also need to do some permission
+tuning.
+
+First, you will need to authenticate yourself using the
+[gcloud cli](https://cloud.google.com/sdk/gcloud/reference/auth/login):
+
+```shell
+gcloud auth login
+```
+
+Then you will need to
+[authorized](https://cloud.google.com/container-registry/docs/advanced-authentication)
+your local docker client to have access the GCP container registry.
+
+```shell
+gcloud auth configure-docker
+```
+
+You will also need to 
+[create a service account](https://cloud.google.com/iam/docs/creating-managing-service-accounts).
+
+This Service account will need permissions to run vertex ai jobs, and access 
+secrets as Admin. Additionally, your user account will need to have permissions to use the service 
+account.
+
+![Grant user access to Service Account](assets/serviceacc3.png)
+
+For your CloudSQL Database it is also recommended to enable SSL authentication. 
+You will need to create a client certificate and download all 3 certificates. 
+Save these, you will need them at a later point.
+
+## 🥞 Create a GCP Kubeflow Pipelines stack
+
+Once everything is done on the gcp side, we will need to configure a
+stack with all of these components.
+
+* The **artifact store** to store step outputs in a GCP Bucket.
+* The **metadata store** to track metadata inside a MySQL database.
+* The docker images that are created to run your pipeline are stored in GCP
+  **container registry**.
+* The **Vertex orchestrator** is responsible for running your ZenML pipeline
+  in Vertex Ai.
+* The **secrets manager** contains the secrets to allow access to the metadata
+  store.
+
+When running the upcoming commands, make sure to
+replace all the `<PLACEHOLDERS>` with the correct values from your GCP project.
+
+```bash
+# Create a zenml root
+zenml init
+ 
+# In order to create the GCP stack components, we'll need to install one 
+# additional ZenML integration:
+zenml integration install gcp
+
+# The CONTAINER_REGISTRY_URI will have a format like this: eu.gcr.io/xxx/xxx
+zenml container-registry register gcp_registry --flavor=gcp --uri=<CONTAINER_REGISTRY_URI>
```

Comment:
This is the only placeholder without backticks. Is there a specific reason all the other ones have backticks around them? (I've never seen that syntax before I have to say, usually the <...> should be enough right?)

Reply:
I copied the backticks from somewhere else in our examples/docs I believe - I can remove it everywhere or put it everywhere. Any preference?

Reply:
Mhm I'd vote for removing as it's just easier to write without them (and also less removing stuff when copying from our examples)